src/hooks/use-ai-icon.ts

```diff
@@ -0,0 +1,51 @@
+import riveWASMResource from '@rive-app/canvas/rive.wasm';
+import { useRive, useStateMachineInput, RuntimeLoader } from '@rive-app/react-canvas';
+import { useCallback, useEffect } from '@wordpress/element';
+import aiImage from '../../assets/ai-icon.riv';
+
+RuntimeLoader.setWasmUrl( riveWASMResource );
```

Comment:
We might consider moving this to an upper level in case we use Rive in other places. Not necessary to make a change in this PR , I just wanted to mention it for reference in case we need in the future.